addon/components/cf-content.js

```diff
@@ -130,19 +131,12 @@ export default Component.extend(ComponentQueryManager, {
       "allForms.edges"
     )).map(({ node }) => node);
 
-    return buildTree(
-      documents.find(doc => doc.id === rootId),
-      documents,
-      forms
+    return this.documentStore.find(
+      buildTree(documents.find(doc => doc.id === rootId), documents, forms)
```

Comment:
Is this a fix for the issue I had yesterday?

Reply:
Just refactoring..